benchbuild/utils/run.py

```diff
@@ -306,13 +306,18 @@ def track_execution(cmd, project, experiment, **kwargs):
     runner.commit()
 
 
-def run(command, retcode=0):
+def watch(command):
     """Execute a plumbum command, depending on the user's settings.
 
     Args:
         command: The plumbumb command to execute.
     """
-    return command & TEE(retcode=retcode)
+
+    def f(*args, retcode=0):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function name "f" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5028678)